src/niscope/system_tests/test_system_niscope.py

```diff
@@ -72,6 +77,13 @@ def test_vi_string_attribute(self, multi_instrument_session):
         assert trigger_source == multi_instrument_session.acq_arm_source
 
     # Basic usability tests
+    def test_get_channel_names(self, multi_instrument_session_5171):
```

Comment:
nidcpower tests with a single-instrument session and a  multi-instrument session. Should I test with both for niscope, as well?

Reply:
I think it's a good idea because multi-instrument sessions return fully qualified channel names.